docs/flow.md

```diff
@@ -0,0 +1,166 @@
+## Flow
+
+- [Adding flow to a file](#adding-flow-to-a-file)
+- [Running flow](#running-flow)
+- [Missing Annotation](#missing-annotation)
+- [Where are types defined?](#where-are-types-defined?)
+- [Checking flow coverage](#checking-flow-coverage)
+- [Flow Conventions][#flow-conventions]
+- [Common Errors](#common-errors)
+  - [Required property](#required-property)
+  - [Missing Annotation](#missing-annotation)
+  - [Type Inconsistencies](#type-inconsistencies)
+
+
+The debugger uses Facebook's [flow](https://flowtype.org/) type checker.
+
+Rationale:
+* *code clarity* - helps team members understand the code
+* *refactoring* - guarantees functions integrate well
+* *code reviews* - adds a static check like linting
+
+### Running flow
+```
+> yarn run flow
```

Comment:
We're a little inconsistent in our docs but I don't think we need the leading `>` for these commands.